plugins/klock.yaml

```diff
@@ -0,0 +1,81 @@
+# SPDX-FileCopyrightText: 2023 Kalle Fagerberg
+#
+# SPDX-License-Identifier: CC0-1.0
+
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: klock
+spec:
+  version: v0.1.1
+  homepage: https://github.com/jilleJr/kubectl-klock
```

Comment:
highly recommend importing all auth plugins, it seems you're only importing GKE (non-blocking comment)

also recommend adding some examples to --help message of the plugin executable  (non-blocking comment)